build/release/Jenkinsfile.tag

```diff
@@ -1,34 +0,0 @@
-// Rook build for Jenkins Pipelines
```

Comment:
Before we get rid of the tag pipeline, we need a replacement for releases so we make sure to create the release branch and push tags safely. Maybe it's a simple `make release tag` command that the person running the release would run, but we at least need documentation and we need it to be as foolproof as this pipeline was. For example, I don't want to depend on the developer's local enlistment being in a clean state. Thoughts?

Reply:
If it's automated with the make, which user should do the tag? I agree we should not depend on the dev env. We just need to think about how we want to trigger the release.
It an issue if this file is removed from master? It's where Jenkins looks at?

Reply:
Before we remove these, I just want to double check how Jenkins is expected to behave when they're removed from master.